includes/class-fs-plugin-updater.php

```diff
@@ -941,11 +944,13 @@ function plugins_api_filter( $data, $action = '', $args = null ) {
          * @since  1.2.1.7
          *
          * @param number|bool $addon_id
+         * @param bool|string $newer_than Since 2.2.1
+         * @param bool|string $readme     Since 2.2.1
          *
          * @return object
          */
-        private function get_latest_download_details( $addon_id = false ) {
-            return $this->_fs->_fetch_latest_version( $addon_id );
+        private function get_latest_download_details( $addon_id = false, $newer_than = false, $readme = true ) {

```

Comment:
@fajardoleo I'd recommend to refactor it to `$include_readme` or `$fetch_readme`.

Reply:
Refactored to `$fetch_readme`.